### ISSUE TITLE
requester.cancel is not a function

### ISSUE THREAD

**Issue by broksonic21** [seantomburke/sitemapper#47]
On timeouts, site mapper fails to cancel.

See:

```
/node_modules/sitemapper/lib/assets/sitemapper.js:142

        requester.cancel();
                  ^^
TypeError: requester.cancel is not a function
    at Timeout._onTimeout (/node_modules/sitemapper/lib/assets/sitemapper.js:142:19)

    at listOnTimeout (internal/timers.js:549:17)

    at processTimers (internal/timers.js:492:7)
```

**Comment by RobertM97** [comment]
I also get this randomly, any idea what may cause this? It's a bit hard to reproduce

**Comment by seantomburke** [comment]
This should be fixed in the latest release 3.1.0 with this PR #54

**Comment by seantomburke** [comment]
Fixed by #65
